phpmyfaq/admin/category.add.php

```diff
@@ -37,6 +37,14 @@
         <div class="row">
             <div class="col-lg-12">
 <?php
+$current_user_id = 1;
```

Comment:
Why do you set the ID to 1? Please use camelCase as well.

Reply:
This is just to keep the the old view wherein when you're given an admin access, you'll be able to view other categories/group even if you're not tagged under it.